code/game/objects/items/storage/firstaid.dm

```diff
@@ -103,9 +121,9 @@
 		return
 	var/static/items_inside = list(
 		/obj/item/stack/medical/gauze = 1,
-		/obj/item/stack/medical/suture = 2,
+		/obj/item/stack/medical/suture/medicated = 2,

```

Comment:
well, the reason why nobody made them was because the recipe was intentionally made to be LUDICROUSLY expensive

and you could just buy white medkits from cargo anyway, so there wasn't really much point in going down the botany route to get them